XCoreV.core_desc

```diff
@@ -426,59 +426,171 @@ InstructionSet XCoreVMac extends RISCVBase {
 InstructionSet XCoreVAlu extends RISCVBase {
     instructions {
         CV_MAX {
-            encoding: 0b0101101 :: rs2[4:0] :: rs1[4:0] :: 0b011 :: rd[4:0] :: 0b0101011; // TODO
-            assembly:"{name(rd)}, {name(rs1)}, {name(rs2)}";
+            encoding: 7'b0101101 :: rs2[4:0] :: rs1[4:0] :: 3'b011 :: rd[4:0] :: 7'b0101011; // TODO
+            assembly:"{cv.max}, {name(rd)}, {name(rs1)}, {name(rs2)}";
```

Comment:
Please fix mnemonics: `assembly: {"cv.max", "{name(rd)}, {name(rs1)}, {name(rs2)}";`

See: https://github.com/Minres/RISCV_ISA_CoreDSL/pull/11/commits/c380c3d6f204bf2efb870ef49da8efcfff263fcd

Of course for all the other insns as well.